src/library/Uno.Material/Styles/Controls/TextBox.xaml

```diff
@@ -377,7 +378,8 @@
 									  VerticalScrollBarVisibility="{TemplateBinding ScrollViewer.VerticalScrollBarVisibility}"
 									  VerticalAlignment="Bottom"
 									  ZoomMode="Disabled"
-									  AutomationProperties.AccessibilityView="Raw">
+									  AutomationProperties.AccessibilityView="Raw"
```

Comment:
Is this something that should be configurable through a resource? Is it modified often?

Reply:
@agneszitte-nventive

Reply:
I personally never changed that property, it seems to be related to Automated Testing. I think it's only there because we copied it from the textbox based style in generic.xaml.

https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.automation.peers.accessibilityview?view=winrt-20348

Reply:
That was an incorrect line, sorry. I was talking about the margin itself.

Reply:
@jeromelaban / @MaximeDion-Work 
On the TextBox default style, padding is used on "ContentElement" and "PlaceholderTextContentPresenter".
The difference with the material styles is that we have the addition of the "IconPresenter" so the padding cannot simply be used the same here.
Maxime if you take a look at passwordbox the revealbutton has a MinWidth value. I think you can use the same strategy for the deletebutton for TextBox instead of adding a specific margin (especially that the placeholder needs the same thing (case of a placeholder long enough to be close to the deletebutton).